### ISSUE TITLE
Saviour Reflections are counted as using Ancestral Protector instead of primary skill

### ISSUE THREAD

**Issue by Warmachine237** [PathOfBuildingCommunity/PathOfBuilding#5387]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Check for support

- [X] I've checked that the behaviour is supposed to be supported. If it isn't please open a feature request instead (Red text is a feature request).

### What is the behaviour ingame?

Mirages shouldnt be triggered by Protector Totems.

### What is the behaviour in Path of Building?

![image](https://user-images.githubusercontent.com/120766081/208221028-01b247e6-1c13-4584-9e92-2acc9c64a5f6.png)

Mirages considered to be using Totems intead of self attack skill

### Character build code

```shell
eNrtXVtz2ziyfl79Cpar9rzEdnAlwZxkT8m32EmceCzHmcxLCiRBiwlFOiRlx7O1__00QEoiZVLWbVKzs7u1lbHE7gbQaHR_3QChl__3YxRbdyrLozR5tYP30Y6lEj8NouTm1c7Hq5M9sfN__-i9vJDF8EN4MI5i_eQfvb-9NH9bsbpT8asdF9gKmd2o4noiin6B725lUgxVmpzLr2n2Og1e7VwN1UEWJeotiNmxPJkEUfFqpx9Hmdyx_Fjm-Xs5Uq92BkMZpPc7lsx9lQSHswf9PIe_o2THGskoGaT-N1W8ztLx7asdtmPdRer-PA2A7uz84sPlVa0LUVJ14X2aqB0Ywt9eXsTyQWWDQhZWDv-AcNCEvFFHcgT_gjQZj0EUZszex5zYzHZtm-w8b2ce3CoVTJnYPqLE7qK9Sgs1gm99NVJJcRWNZq2hfdbFdZGp4zBUfhHdqcMsKg6HMvFnnMLeJ528bfRsn8OwFnGcj-Miuo0jlc200UV_-qgBVywYv4yPLgYzqdi2KXQHlCYoJov50mKmrS7KT1ExPIhhRtZoRfOe3SRRodZgvkijPE3qjNwVeF-41CWOEPgJvqbpcSTA9pArqLBd4S7q8ON2HUoZAzt0qMCc0ad02mAl2BFon9uUCad7rIfjOIaF3OB0bbBDQQTDwnU6R3uYjjxwA43ZEZhiivZdRAh0vIvzXCbyMM2XMAFNeaEycCDFSgwD5afgc-oshOzzJZppZ-5s710UquUpVxpKxbBqb9Ybx_FgWbqVBa_XoUtYQ8tRDoqaZxPdAtX3BiXvtIYj9WNG5toLBDYouwWeJbNhYLpIYIOyO_Ycqbu0gBj9tHKMZzg-vZi5I8dmYp8Q1ybUpd0ucPiQR76Mz-WPaDQeQWC4kt9UUvOklLjdZngzLBLwKl3cxHZJp185iTK1FuNhGgfrMQ5lmndygicTixbclBLo8AJk4b_Q1GeJP1MioosEf0wylavsrgZJFjWhWS5hhWkA5MVqaZ5ZM9VKrfUPLW7sRiVViw_LOad3SvnD14D6LmWhlvPmUyrbXaxdTVzX7kKpLbrF3QbS5HikJkdDCsGpoBwC3yIh7SoDAQubXlFrx4nKbh4Gw0jFNcvBy5BPunYob5fgNGqvszeMmy_X4Eo2VGddUSufZBYsF3ZW7FMJvjRPLd2AFGAxcn6vissGzx4mdJ92MgH9dM3VeBY2BDxTo1vK2IDh0eQsMSl3Mq8HI6Jh42KbKTka5oI7feG5giQAWAI1l5w43eErS7_qJCteka-fjdJxVvP9giy2_pKhPhAXPRVTywThUgVjvxHE7c54fxBDkjw3iO71BR2L4zaWzp71i0L6347S4EYty2IaWZ1jML69BT-qZ38-0ySLAMGlyqMaGHT408QfYPnW_Vi3wjRwWLqBGfHyDUwB0dKtzHE8aspeiGXmmrHZEtTzTXQv90n94hwcpK59mKrJeVqLpPvdiOME8s16xogWEjZT2u6Vld5D54e6fJWvRg2obxboOruSqeT3h6XlN8iXauA4CcaZXg1LtzHPsVQzfTNtpnD1LhpFxaOCkK5ixbA6j2QhrVzJzB--A9N4tbNT-3Qi49gDdwHfaq6Xz00dUf91NrpNs8KKZV70fT8dJ8WpzIevdpgtsBf4to8ooizkthKhbduKelTy0LFDEVLf9bwdw3qpZDyCWTosP-oBSr9QWSmL2g4niHFhozBkVEqXilDZmCAUkjD0kO8ojOwyN3l5KGM_N0M7S27HhZWYAuQoyv0v3jgMdZ0RhlZkplZ6fHJyfHh1dn1caaPOkn-L4vhLMh55uo5W_leXK0vKgTKe3MrHXl7--WrnOlL3A811pAoZxTCnfhrH8jZXwaudUMY5tBzBn4ZmAGr3iwXSDBXkJpPiUrusGUG3pOMfKitgvICH_CxSnf2aPn-iU2WDGiuVNdB2abrg1y2ojD-HevINluvQlKnOdkvR1c7O4eiHC3jBdGXc2XL19AlNFA-3KgdQEIWRr1fd4im_AuqSaoFefB9Wuf-wYL6rKNotw1RQuwSUD7uZy4pkF3f1dIFWTQW2U6vl0272I-XLzrGXD7uZp-lBmpxqV9cuZUq1QNL7qqYLi6YfxTridc7scaymJN0CPxRDlZVhtFPSOfioCcnChZNF3rjoXsY1igW6MmWMDg3pZ92sZYreMQb9bIEnauSpHQqt03SLKpO7Tke2iPVygT-9XOxKJ9lCh-bLpwvGP8mbOoZePV7QdQPcOqe-QoGLXAPkCN2uQT_sZi6X1mMRRTZeUoJZBkcqVODAFq6DKc0CB1yMkyPwlcUChS4pqgxo_bs0CsoEZSNpZpDbFVY6mC0pzqRwmyp_LrPbygRUVdgO518jeUIQeO3TBbhgOUnTnOcUwKneqEvjzQQ-qjZvNM60yGUSmBrU0gN9-XwCjl8aPJJbOYD312qUHzxA0nWiPe7cJp80KqgAa6HThibPZPEHKpTjWH__y1jGUfGgE5nat-_KIwdJmo2mOwwgapjeayMqJWp4lIMTffeufNKPi0qYbm4yslIlVYfMMLH5rvrSHDDoz3ptRmz6HSV-PA7UWVLlo9CbKNYJh6d7pk9Q6Br61J3lcapLiEreguqJNbjXifKc8FKEaf1vL6GTlYzXcerJmEwaKIdO0E7jMZ42pCWdGXgTRtpjHMgk0SnH93Lw1SOtyB3rRo3053NIMALI2J6fQWKXP9f6eW56BH91iDETMjdGk7BV3dRJj65vzfphVRKel-otG3ha1UbaJsrGP0PZldmVWPoqnaDeVXU-Nd534AdVNidmeZ2X_bCK1Koj7C0OtB8EkXYdMj4ETxak95sMttzV_RQlwRpDLZktw73sKFce5K8XGwyvVczyAzxOYl3Km_jvn7FyDtLgwZpg00frBi-5arD7lJ6vo1tdXsqib2pt_9Qmo27IT2jXsFsTftOz8ygpQcl4NEohcl3AYipkPlrWvPjy5qUCUz8tI-tFPM4bXdCjnYTJNu0Y0guZlRTL2mJLm6urrX-vYYAKLCPQMhKtSWmpVYuX4I2n3n97jsgcTssfT98foLtGU6urrKun66poWTP7mHjQn2BSVfgZVtba5AZGVsmzZsWTn6i_a8i8zmUS3Y5jkxz-DAW2t7mBBrVAqy7xp6rwSMkgfviZFtjW4gbqK8WtZH9Lh-klcokcArGvzzPDUF9g8eJqqKyBvIvg613rnbrR6dzgPtUVrIVIeINg_rRD_phE38fqPNKHpz_JLIvSLF8D7mw9QpzIvFBZuUGwAYprFfNqx1lyWCW7VfLnWx-kCcAT4esPslXM8nOnjyArvRVmXaVjf_jzAOupgoVZ_OEJNczgKEoq_7l2Pt0mZZV0us7_Z9dwKWJpFV9HhVHHBtnAnICmYqtqVlcqUPH-tcz2dSb99dOrJvfyZlrybc3NTfYx-0lgNmn7Y_2u0JpjusiUH-XrLb8Z78-uY5E_2E4O0yxLcy17mKXjYH3txvJmrA6UzNYqF5bsVsW_vTgZm9jUGFU_BlcKjlThpQY2jW6DQt2uAydn4dFI-PcyIPyUhk9iGOiRzIdrW85jCcubjeG1DPNPLJOlOhf4Y0PildJnomQM0CxK8rV12yFmeQVPBFiVhK2tzE_DKNMvkB3EMlDN1Gqao6w76nnZqyPaSoDV0rvOCuGf1vRWmxhzQhDGfpitj8faZKxS-i65Lc2-7ZTpY3YDufyHLFDZJilTq5jlh1iyWxX_T98b-8O9lz4qao7fr21ChtvIWUa9uFG_0ecECu20LjIQ4Bdp9t_ywn_LCyuvFtDOncr_WPh0LfVeahwMEnm7fvrbJmR5BWt2S_NbRsCWt1P16dwPSeNszvqW1ClrhU1ykGHdD1VSbVxZtVMyW3F-B3GaBuXpwDXn87GEFdINw2zNTiAuYaRPWul5CjlTMhiqOF7fobfIWH7WSm6rZH_kAao_B6owB5TKc0n6z6tMqcnxIyMHV0d_4EP9gpEzcwIIVrr2pw_V-dlXO_-0ieB0lyCO-b92_8kYxfYuwcwW8IkIF9FdyrhN9SeX7DLHZgT-tpGgDJ4wgvQnzonY5Q5ymJZBqI13sUMc-MAFRWSXY8z0J-YKgnaBCbN_wdjTQB9kosKmu9hmYpdhW7i7FGEMRIaUY6L_FciG721C8C5xbG7vlmJt7rj2LgNSoZ8KZ9emhGN4SogLTwVhMBZis13otc13bWzrsXJiI2iFUmFoYLyOAxJsW8DYOXEw2-U2cYHehY7YjCMQ5rjuLuPcpbuucBzoqrDtXUpdDmrgAhplrovQrkP0YFzugjxQlu6F1o2NdCccF3rIoVmtIXtXIISBmQi6a-tbN6AfMDr4F3MQARreLScAOgnKtpFDnV0XVAdjEpTvUkKIoxt39Uw40C6zGRK7LkGgFIcJpqlBlDD9wFoIdNLRzTPq6u8dtKtnlbvQP_i31LVra20KDjNKYd52MWG41DWBNk07CLQNg3dAm1ybxy7GAoPeYeqo1iPI0R0XrlYxcZhu2IbJK82GwWhd6ATVo6IYus9A6wS6wkCbpQkyMAJoWYCGCBUCtCIceEg4gn6VFsYIRtA7gQX0xeV6rpFAjp59qo1RwCQL19W6ZKAtx4aeu0yrwpg89F6zM22rrh4tdEtPLJi-VjnX9oW4HrMtmKstEbTgUg6iMOW6_wiaZMSFJkGhCBBNAWuxdkeQdmj-ZO3Z1em7j5fvSp81LIrb_MXz5_f39_u3shimofoRxWrfT0fPb_UtQHdqz7inPS32eR_-d3Dz-cC_Eb8eJIcR-fWTur_9_ezkw7vrr7_i_PDDh73MeXh_98W_-vjjyDkMj64_f_9BbkYP57-rzOM_sr29L0h-dc5-e3P76WuUDqKv_btD5-BHeHXmuP3wCztx-vLr5ejtr8f98JfPyZX7yf3-tf_dG6TFPd_DB2e__3Y9oPTr1yN5-uBcn519kTw8PKeD69MfUX5uf2Lum483x9_s915y_eG3z1-_ZBfq64EIv4yD7-nFb-h9f3wmxHWf-ZF99_HzffArINery08PHiHZ0Wf_3i5O-3vZ-c1JcJUcDD6rt7GSl-zyXsrfTk5OT7Ivau_N8OqAj65OH65c9_OnwU3_E7767S36EX5RzrvDC-HfnL85K-wQJ_e3vPC8e__goH980x-cnfT7x9_7v_RPDvqjb6f5W_-HuPt6VzgPX75Ep316lIyH6cGbs9-QI99c0bfIP8j2YnXd_-p-_viq9MeT2XtZ3s2UV-Gn_GQcmplrvdYAkhVlzCDTMDVHZ1bQjA530ulVN6OjXXTaXz0WB9Fi1ln4AIHBxBAdOfQf7yGDMM_0l5MPJq7oF7Wq19sGRaZR5O9pOvr8aoe5-5yBoygPoJ4qWZzL2yqiaZLqGKtbHV0tZHEUQcTJzPnmSRjUhL--2tkDb0b2YdXb1evaL01orWJafQt9nKvqRJ1J8szXE6BqmOrHXC9lBuH3hXXZvzzuvR6m49i61mcP495gJAFKH8Zjk1m8UffwXbnHa50dvbACcA9KOKEDLhr5DleKMh8L8AROwBCnIcKSKxQEnpABdBoipxTU922KfKwI7ZmeGA28sBzUM39dqu8vLIZ6Z6PbOPKjIn9hkd4__UyGhQr-1Q-C3CLWRbnqrTK4Nx4rQK6m200a6yaTSfHCwvzvOgHIlMz10SVzwK23gE3GeTrhfUb0cco-EM1eiFmaF2PDvFqD9O-aqTxaVb6JoQ919nB15qrBakW5dSCLIlZ7pzIL9KEFY9PnGj4mgG0z_a--WIzvVNNfWX0nBXmSgj5JwZ6k4E9STF5UNlCyacOkxYY_eHF0pw9EHI6zKO0dZtEoh0-P7JdD-HUB7YSc-EHAXU941JYBR7ZPfM9xkZKhclyfYSeEJUzckIRKEkYDsHoRuA37tZ26yaLeM4z03IE3UMlNMbRkElhnSQGQNbpRZhJR3RLL_KOHWf3LEoJbk3f-J0Ron-K_W2k4e1Am1JMkxryPBuwyt_SLV38iK2ibQdoyg4eQN_jxQz6yPulTX713-v3kbkfEOMBhEnjgcQA3Ceko6sFMBQ5SPkBV8ESAYTmhgS0Ug6fY8YUPQExhoLGV03RErOaIOKvPKp1zRKLTEZG2FQqLO1Nl760q0KzkukjdNurnLbsdwmuwfesDYKZuElNI0dZ0CLEmLh66KT8m9_LOVF_iB-v4Lor_A_wLa1rn-w-X5_13vdcaq0ICKpNvdTsMCfYpYY6UkNGwELI1yNMC6nPmEup6yKcAuCm3fQpRMnBDGQodIiFN8wLlUdSwQ9yr3lXR3qQyF1ge1oF1WbdJ3KOoxWfoLXLjGspzzVb5OsBSE9amBr6KGjwS-CFyQ86ZCgWH3MDzqMTcB1_KIZVSxCOI-thjQcgcl9tMEYQCpRTHDpJNNdBWNbzee7138PPVYDfVcN5_fXbYO0p1Vfl_AGjl_5tbv4wj_1sexbBQrBPQzze9tAb6ZXn4f11LCmNI9j3qIEhX7dCHOAP5lMNd1_WJgwLI95jvSxp6DkMSBy7ypBf6no0VC2mAm06roaaZ_6K8GZUIBA5_-mbIjYwSS1bdLC-BKKthD-nYChSoTc5rsEfshhdq1a9V7dmUtZOlVnibtp02bc82hNI4zXNoWr8hPFN1MVRWPymiIv0RJXV9EzvgUnHkBw5xKfGYi20wSogIihMeMIwkDYRyPNfHYUAl4iFYMfJsh-HAdoOmVbKZvrFbVzhqKtwGdWX6lh6NNke6nGVVt0IBPCv1o11rkhZANoLvg3IGZjeFleMDmhNwwzOmIFUl2y9jNVa91zpG5NbZaDROoFez936sMM0sG3ITnQ7kVhTq5EA3Ahl0RbI5UFwQ3kVLeH-b6TKrdZrGoNUM-ng5tzZCbfeB7YUBCYNAKg_chHR9agcwWz6WKnS5vrYDsg4KThcpZgeuC8iMOVgJTubXRldAr2UWnwGXg_lG5iZTCIr3KsutEFK735UFWHoUgbq1KtE-mSqzVCJY3IOJ7NLMDCjXe4BFVknylByZLCG3TAbA5uGcSQvKDZHeM2JSjFH5OqNVvrhulW-uzwHGUqHa4sskMDRA6Rm2TcqgUcRxbG7WgYZmqUM-pZheDlRPLJ4RVGYcegNAI9b51KMGWJ5xWu_rnwxpbgMPLKBwnqQQ3UvCbfNrA6WKoZ6OI8hdEmX0OfNqkHSqu8gc-KyvE0x8WBRg9UGAfdulXBIhAsZsDCFDSu5K4tsIYocnOFYOC5Ttecj2CHcdSeYzGNweQ-xVXNpZog1l-s1Dm2MyF4booer1BGlSu2-aULVJ0AfVxrd6qZU7HAsF1ZflHONPM9g2M8CozQ6uQEJ-o1-Cn2EKrYGoUM0gdwEtpXEzxkEYoyETyJE2YQBBIf2RrgyJywFluAHyXBmQgCnKbDskKAgIYA8eSJdS4TmYN-xBkJk9kEZ1xm4ir6lL-JhPwld1oxTYiYTkIhzHcU_f9QAucB5maMd5GhV6cmSiPd7oocdbijRrYIr1AhbGbdPSTyTko8VkRt7IoDYdF5HK_PkQhlgY-MxBkJmKIGDI4yH2XaQCSDgVYDhOQs8XwhUMS5uGyoMkNnQ8QIEq8IRoTAUj7UuTiDl411Dc5C6LHmXbxWybmf1cyebj-7NfPh6D6m6ieKLeUwlznjz0rsbZ9zFgFIB0o3GsirqCIcGingxC22ahCsHAeeBzKhxMEZdcgBaZ57raMUJCJgLJAid0IPOXgPdsx5vLMnhdq7Sr-hjHqS8LsOurLNLpxckYPNwz7GqfdKR-FMrE5EelHlhuy-cmzyh_FFefMbf-nUaEPdaAA_ruEtOwuTBl5n5LWNNsKQfwaKCjgfjQ7UxOCgn_aQG81UDbK1L6dabEepemt72r-3RvUABwfgRcCQRcyRjzOVHIdsBAXRYiz7E9SIYppLqh62Hphhhhm_jgkiELIUEYusoWfijmgKuoO12xCLieq0JBDKxgqzYKCBcH48wAPP27GEkZJOk0RsJYwO2C8XKD-PSVl8Z-2nBhhV-FJsSiRIct4NVBjw23Er8Aj-paZhcenYHN-W6aPtTl_AfYJWuxy6NM3uhX_wqZ9U7isUE2ModgBM6yDJw6eLPqb3g0udsfkBFEjH3KwLCqW6JKyupDB2nN1hHxBFU-gA0Ggc3xbBEoSNoc5aoAgUVj6ejqqvIEJjhQgfIJV0T5GoNg4qmmrTfBRq3edamLPfUMTnSUZJtn68E3hkAEwJFacnr9glUSmfc59eWNec-pe1GTal-axNtETh0NoSFw4ibTqqq8OkmaAGDDoX9jwKD0yUoQ9SrL5FIjy1xxNPG0zxz78WqhZGbljbSMLbVGDI1WlOmsUaHGT-mxpU_fzCjdJrxSUtNvNfD_-yRoCyjcBWuRt4KYgyhJJxjmbVT4GuS-TfTUDmI51ld4ZNXnOpAJBCA_EbiBQK7rhS51KdYBBNZVQInwbexJQpHDQ48LQbgNIcYPuWLI9n0vbCZxtH0dwRqyDjpX0WrV07KYwQ1MIQB-5venmPM03JzWITqamv3iUO-Z274L-gF0WxVZ5LSSVWYQu1aiZAYpxXFVudFZxrTYpeNH9RwQnX5sLilXmQ6aBKEy99aXgpTB8q-CihbFFrsltlyWm4n6Zazep4dslMP8AKrQ7x1MYwvlvDO2uILbs9hCidMVW3AjcQKcBBbvYLB6iRjWh6pC38MCA1ryfMkpRZBMuQ5iJMQ2A3KPENu3IbzYbK6m4fDu5XC5d1lfELiJr2omgfaJ2W6dMwvz5_QO5Sqx7VFe9_3m0j7LU9pIy6tKAwDzpL4zDNGC1Y4GPLMN1K80ZV1KvXXQc1BLRqztuKLTQjtCB29fPc0FP7mG2zL34_63jretMOG0LKvrNCvUD8gk_G-9QfFwE8nEuoYct7E55DqCYs8LOEG-QFQR6ng-IsyBJeArQoTtMlchL3CFZBQAlsOY7SAqkKBUNI_WiJrh9zsLe7h3KnWNuu89mG2VkgUMCzWt1WGPcUuFTx4fVGlsWc8SgCp5uAcwNjm7ML-DDll1sxTdnpj89XegsWiFGrULQHr1-z8aVmQT8KOBtCElRT5V3HGI8BzXR9JxkcuE9Jh09ZFWIkWIAsDxOLDB1GjIlB82t7xwB7K43Gvic5s0HWkDDEwm23gZ81qPzkJzayjvlDmbNXOdJ7BMovChd5VFNzc6qy2RLUEw_2Fc3WBpyojH38fR7S0IxKxR6JrHJeXhMWPO7NE2TA_jzo42H3UAmmlpeVOb_HdIRd0Wv3ZgLqnTxer8toYXXstxUsSqhhkw7sYM-qzkDDNoys581JDWrN12QgeSSycIpYclpYGLmcTSRcThnCskeEADipWvEAU4oYvfBAADUwLSWOTP1V5wu7Uf7Gl7rwMH5jbt_Rl_OvmrLKc631P-ZCq43lqeWsOvDwYBPyVytsNIOgs0vKVAQ52Zm23PKulTW4J_fXsnqNUHHw4VrHkvGk1yvnMw-DDyTdEX_NZ4dKtfC7beyRHMlMxmK4AI0rkEBOK8tgRsjLrXAEADt-HxhcKB4_gCMIBNdCqpHIdhHjgB51jygIeIuZRwyRwlfCk4JRIh4QPgYMrzm7iBdXv8y72Deb_vzm0Tut3H1QR-Cs4K55GpYtSArEValhLLQncdl4dZWrKUJyH_vOcf9X3DXUe1mxcRx2lhTqtfeB8v3-nj5eUvikze4dXX284BN4vPzrejSf-ekjMvg6who3asXjzFY3LJbXS8TQGrdP5AxcV6_OXh-8lJ_VJauTFYf18BLzkQ0jYQuoWZXEOp5I-QwbYwFnsL_VhHRtvc4FWMrM3aV5nc8rKojbrQYuf2xj1YZaVNXqSeLg13vTbplrwE3nj0bAvmiLewLNiKdlCbA2e5WduoxTbj34bqNnXTNa_mrDWITeMMXUmL02u5NzGh9uncfCnwjSWs7oxqZjzNE0phZwGQz79CWLKWL2HU3iqciF4koHy3sCFg9rrhvICOPpc2V9O0u6jJ6vXIepO1NyaXbLJ9svl2rI5s6gLsLXg-ugU30iqjbUrMm6ONGZm-S7rkhLQ5AbYlPLKpM8LbMQu6oVOuqcNeh7-mBryWHraVhKxjmrW-801swd50bW7JGLZl22uhhfKXXyZ8bGkX0-Tjm9jAKvOgD5Q1LICu597JdqZuG955TSuq3QqwJp6hq-q91lOxyYSz7Sifb4zoVvYANcOzN1mqq3S9PNpba5nUi2WTe3zMWy7m53jTJIxuHv24rn5H5-EsP0jzfPZrvBdRkkh_-qurdfpbmRRDlSbn8muavdY_PD1huhqqA_hLvYVPLYyeTAL9W6AT8n4cZbKFznRoEP2uLvSbXDWGcxVE41ELx2h65qU8qtK_1R-CS6ULvvHD7CeDJwui2a9xGH5Mhmn8cK73YXYsL01jJZMqPD7d3IfE7DbMmnEXqS1K5tT2Pk0Wt_LebGlcykx9yMqSdXU6Z9YiXihgcnZvYH7oV794UuOs_Qq6r0ALgcoaE2F-93dK7iC0mF6fo7jQd4pNOJ6Q3_jJ9AkPfaKR6Zb25GdzphNMyROsehOy-tnepbsI36wyJK2C-RFx9HQbqw9Gt7Q612QDdz3lzXM9NVe1X0yqqUMsoY9VdWi2VGbkgi1pRKvM7ZSpo3Mvn0_c7MvnF7IYfgjND7QDwz96_w_x--11
```

**Comment by Paliak** [comment]
This will be fixed in #4599 but i can make a separate pr if necessary.

**Comment by QuickStick123** [comment]
This is fixed afaik.
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/31533893/250f5123-cc4a-4ea8-b1dd-50960aca8d82)
